index.bs

```diff
@@ -156,53 +160,53 @@ spec:fetch; type:dfn; text:value
   </section>
   <section>
     <h3 id="policies">Policies</h3>
-    <p>A {{Document}} has a <dfn>feature policy</dfn>, which consists of:</p>
+    <p>A <dfn>feature policy</dfn> is a struct with the following items:</p>
     <ul>
-      <li>A set of <a data-lt="inherited policy set">inherited policies</a>.
+      <li>An <a data-lt="inherited policy">inherited policy</a>.
       </li>
       <li>A <a data-lt="declared policy">declared policy</a>.
       </li>
     </ul>
-    <p>An <dfn>empty feature policy</dfn> has an <a>inherited policy set</a>
+    <p>An <dfn>empty feature policy</dfn> has an <a>inherited policy</a>
     which contains 'Enabled' for every <a>policy-controlled feature</a>
     available, and a <a>declared policy</a> which is an empty map.</p>
   </section>
   <section>
     <h3 id="inherited-policies">Inherited policies</h3>
+    <p>An <dfn data-lt="inherited policy|inherited policies">inherited
+    policy</dfn> is an ordered map from <a
+    data-lt="policy-controlled feature">features</a> to either 'Enabled' or
+    'Disabled'.</p>
+    <p>The <dfn>inherited policy for a feature</dfn> <var>feature</var> is the
+    value in the <a>inherited policy</a> whose key is <var>feature</var>. After
+    a <a>feature policy</a> has been initialized, its <a>inherited policy</a>
+    will contain an value for each <a>supported feature</a>.</p>
+    <div class="note">
     <p>Each document in a frame tree inherits a set of policies from its parent
     frame, or in the case of the top-level document, from the defined defaults
-    for each <a>policy-controlled feature</a>. This inherited policy set
-    determines the initial state (enabled or disabled) of each feature, and
-    whether it can be controlled by a <a>declared policy</a> in the document.
+    for each <a>policy-controlled feature</a>. This inherited policy determines
+    the initial state (enabled or disabled) of each feature, and whether it can
```

Comment:
'Enabled' or 'Disabled' (seems better to spell them consistently (or change the others))

Reply:
Changed to be consistently capitalized.